src/V3AstNodeOther.h

```diff
@@ -847,7 +847,7 @@ class AstCellInline final : public AstNode {
     // except for VPI runs where it exists until the end.
     // It is augmented with the scope in V3Scope for VPI.
     // Children: When 2 levels inlined, other CellInline under this
-    // @astgen ptr := m_scopep : Optional[AstScope]  // The scope that the cell is inlined into
+    std::vector<AstScope*> m_scopesp;  // The scope that the cells is inlined into
```

Comment:
I tried to use List[AstScope] here with astgen, but it just seemed to produce an Optional[]. I didn't see any other use cases for this ptr/List combination, so I think astgen should either support it or throw an error. A couple questions I had about astgen:
- does op imply that those are children in the tree
- what's the benefit of ptr over just having member variables? I'm assuming it's something with broken links?